tpu_inference/models/jax/deepseek_v3.py

```diff
@@ -92,20 +91,10 @@ def __init__(self,
 
         self.random_init = force_random_weights or self.vllm_config.additional_config.get(
```

Comment:
please avoid using `additional_config`. for this use case, please use `vllm_config.load_config.load_format == "dummy"` instead.

example: https://github.com/vllm-project/tpu-inference/blob/ac26352871e9d0ab759a2af5097bc419802a98dd/tpu_inference/models/vllm/vllm_model_wrapper.py#L109-L110